DNS/Server/MasterFile.cs

```diff
@@ -25,55 +25,55 @@ public class MasterFile : IRequestResolver {
             return re.IsMatch(domain.ToString());
         }
 
-        private static void Merge<T>(IList<T> l1, IList<T> l2) {
+        protected static void Merge<T>(IList<T> l1, IList<T> l2) {
             foreach (T obj in l2) {
                 l1.Add(obj);
             }
         }
 
-        private IList<IResourceRecord> entries = new List<IResourceRecord>();
-        private TimeSpan ttl = DEFAULT_TTL;
+        protected IList<IResourceRecord> Entries { get; set; } = new List<IResourceRecord>();
```

Comment:
as [Design Guidelines for Developing Class Libraries](http://msdn.microsoft.com/en-us/library/vstudio/ms229042%28v=vs.100%29.aspx) mentions, I thought it'd be a good idea to convert it to property.

`Public and protected fields do not version well and are not protected by code access security demands. Instead of using publicly visible fields, use private fields and expose them through properties.`